khard/carddav_object.py

```diff
@@ -1171,6 +1167,10 @@ def _process_user_input(self, input):
             contact_data.get("Prefix", ""), contact_data.get("First name", ""),
             contact_data.get("Additional", ""),
             contact_data.get("Last name", ""), contact_data.get("Suffix", ""))
+        if "Formatted name" in contact_data:
+            self.formatted_name = contact_data.get("Formatted name")
+        if not self.formatted_name:
+            self.formatted_name = ""
```

Comment:
From reading the diff, I know this is supposed to trigger the `property.setter`, but if someone later on reads this code, they will probably miss the context and think this `if not x: x = ""` is useless, or worse, remove it. Maybe this could deserve a little code comment?

Reply:
good idea